code/style.css

```diff
@@ -1,4 +1,139 @@
 html {
-  background: #0025ff;
   color: #fff;
+  font-family:Oswald, sans-serif;
 }
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  background-image: url('images/coffebeans2.jpg');
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+  
+header {
+  background-color: #a97559;
+  color: #fff;
+
+}
+.header img {
+  display: inline-block;
+  margin: 1rem;
+  height: 5rem;
+}
+
+.header h1 {
+  display: inline;
+  padding-inline-start: 10rem;
+  justify-content: center;
+}
+
+.navbar {
+  display: flex;
+  background-color: #9c5239;
+  justify-content: flex-end;
+}
+.navbar li {
+  list-style: none;
+  display: inline;
+
+}
+
+.navbar a  {
+  color: rgb(15, 15, 5);
+  padding: 10px;
+  text-decoration: none;
+  
+}
+
+.navbar a:hover {
+  background-color: white;
+}
+
+.main-news {
+  background: none;
+  margin: 2%;
+  padding: 5%;
+  border: #cd5c5c41;
+  border-style: groove;
+  text-align: center;
+  display: grid;
+  height: auto;
+  width: auto;
+}
+.main-picture img {
+  padding: 1rem;
+  width: 20rem;
+  height: 25rem;
+
+}
+.main-text {
+  padding: 1rem;
+
+}
+
+.small-news {
+  width: 100%;
+}
+
+.cards-small-news {
+  background: none;
+  margin: 1rem;
+  padding: 1rem;
+  text-align: center;
+
+}
+.small-news-container {
+  background: black;
+  color: #dbc1ac;
+  display: grid;
+  gap: 0;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+}
+
+.container {
+  background: #9c5239;
+}
+.small-news-container img {
+  object-fit: cover;
+  width: 100%;
+  height: 80%;
+}
+
+
+.card-ending {
+  display: flex;
+
+}
+.card-ending p {
+  margin: 20px;
+  padding: 30px;
+  text-align: center;
+  align-items: flex-end;
+}
+
+.footer {
+  background: black;
+  text-align: center;
+  font-size: 1rem;
+  padding: 0.5rem;
+}
+
+@media (min-width:0px) and (max-width:667px){
```

Comment:
I think that just `@media (max-width:667px)` is enough here. You don't need the min-width: 0px;